extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+# DoubleClick
+
 ### <a name="amp-ad-network-doubleclick-impl"></a> `amp-ad-network-doubleclick-impl`
 
 <table>
```

Comment:
Let's cleanup the Doubleclick description to not include fast fetch implementation details and instead point to amp-a4a documentation.

Also change the Availability to launched or just remove completely

Reply:
Done.